### DISCUSSION TITLE
Can I use this library to convert PNG image to SVG?

### DISCUSSION THREAD

**Discussion by zydjohnHotmail** [SixLabors/ImageSharp#1700] [Q&A]
Hello:
I have a small requirements for converting about 20 PNG images to SVG images.
I want simply convert PNG image to SVG images using the same parameters, let me know if it is possible, if yes, please show me some example code.
By the way, I am using Visual Studio 2019 on Windows 10.
Thanks,

**Comment by JimBobSquarePants** [comment]
Not natively no. There’s a 3rd party library SvgSharpie that renders svg but I don’t know whether it creates a file. 

https://github.com/UkooLabs/SVGSharpie/issues/2

**Comment by Ryan-Millard** [comment]
Hi. Do you think you'd be interested in having this feature at all?

I currently maintain a library that converts raster images (like PNGs and JPEGs) into SVGs. We don't currently support C#, but I definitely want to add support for it and having a consumer would be very motivating to get that done. 

If you're interested:

- GitHub repository: https://github.com/Ryan-Millard/Img2Num
- Demo app: https://ryan-millard.github.io/Img2Num/
